.github/scripts/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+
+# Import gpg key
+echo $GPG_PRIVATE_KEY > private.key
+gpg --fast-import private.key
+
+# Deploy to Maven Central
+mvn --settings .github/travis-settings.xml clean deploy -Prelease
```

Comment:
You have renamed `travis-settings.xml` the script won't work.

Reply:
Fixed